Fails to build on 32-bit FreeBSD: i386, armv6, armv7, powerpc

Regressed by #620. Found via av1an-0.3.0. From [error log](https://github.com/GuillaumeGomez/sysinfo/files/7824763/av1an-0.3.0.log):
```rust
error[E0308]: mismatched types
  --> src/freebsd/disk.rs:65:39
   |
65 |     disk.total_space = vfs.f_blocks * vfs.f_frsize;
   |                                       ^^^^^^^^^^^^ expected `u64`, found `u32`

error[E0277]: cannot multiply `u64` by `u32`
  --> src/freebsd/disk.rs:65:37
   |
65 |     disk.total_space = vfs.f_blocks * vfs.f_frsize;
   |                                     ^ no implementation for `u64 * u32`
   |
   = help: the trait `Mul<u32>` is not implemented for `u64`

error[E0308]: mismatched types
  --> src/freebsd/disk.rs:66:43
   |
66 |     disk.available_space = vfs.f_favail * vfs.f_frsize;
   |                                           ^^^^^^^^^^^^ expected `u64`, found `u32`

error[E0277]: cannot multiply `u64` by `u32`
  --> src/freebsd/disk.rs:66:41
   |
66 |     disk.available_space = vfs.f_favail * vfs.f_frsize;
   |                                         ^ no implementation for `u64 * u32`
   |
   = help: the trait `Mul<u32>` is not implemented for `u64`

error[E0308]: mismatched types
   --> src/freebsd/disk.rs:132:41
    |
132 |             total_space: vfs.f_blocks * vfs.f_frsize,
    |                                         ^^^^^^^^^^^^ expected `u64`, found `u32`

error[E0277]: cannot multiply `u64` by `u32`
   --> src/freebsd/disk.rs:132:39
    |
132 |             total_space: vfs.f_blocks * vfs.f_frsize,
    |                                       ^ no implementation for `u64 * u32`
    |
    = help: the trait `Mul<u32>` is not implemented for `u64`

error[E0308]: mismatched types
   --> src/freebsd/disk.rs:133:45
    |
133 |             available_space: vfs.f_favail * vfs.f_frsize,
    |                                             ^^^^^^^^^^^^ expected `u64`, found `u32`

error[E0277]: cannot multiply `u64` by `u32`
   --> src/freebsd/disk.rs:133:43
    |
133 |             available_space: vfs.f_favail * vfs.f_frsize,
    |                                           ^ no implementation for `u64 * u32`
    |
    = help: the trait `Mul<u32>` is not implemented for `u64`

Some errors have detailed explanations: E0277, E0308.
For more information about an error, try `rustc --explain E0277`.
error: could not compile `sysinfo` due to 8 previous errors
```
Note, FreeBSD itself [supports](https://www.freebsd.org/platforms/) i386 on 12.* series as Tier1 [until 2024-06-30](https://www.freebsd.org/security/).

--- a/src/freebsd/disk.rs
+++ b/src/freebsd/disk.rs
@@ -62,8 +62,10 @@ unsafe fn refresh_disk(disk: &mut Disk, vfs: &mut libc::statvfs) -> bool {
     if libc::statvfs(disk.c_mount_point.as_ptr() as *const _, vfs) < 0 {
         return false;
     }
-    disk.total_space = vfs.f_blocks * vfs.f_frsize;
-    disk.available_space = vfs.f_favail * vfs.f_frsize;
+    let f_frsize: u64 = vfs.f_frsize as _;
+
+    disk.total_space = vfs.f_blocks * f_frsize;
+    disk.available_space = vfs.f_favail * f_frsize;
     true
 }
 
@@ -125,12 +127,14 @@ pub unsafe fn get_all_disks() -> Vec<Disk> {
         let is_removable =
             [b"USB", b"usb"].iter().any(|b| b == &fs_type) || fs_type.starts_with(b"/dev/cd");
 
+        let f_frsize: u64 = vfs.f_frsize as _;
+
         disks.push(Disk {
             name,
             c_mount_point: fs_info.f_mntonname.to_vec(),
             mount_point: PathBuf::from(mount_point),
-            total_space: vfs.f_blocks * vfs.f_frsize,
-            available_space: vfs.f_favail * vfs.f_frsize,
+            total_space: vfs.f_blocks * f_frsize,
+            available_space: vfs.f_favail * f_frsize,
             file_system: fs_type.to_vec(),
             is_removable,
         });

